memorywipe.py

```diff
@@ -0,0 +1,496 @@
+            except subprocess.CalledProcessError:
+                return False
+        except subprocess.CalledProcessError:
+            click.echo(f"{self.partition} doesn't exist")
+            return False
+        return True
+    
+    def _veracrypt_interactive(self):
+        click.echo("Volume type:\n1) Normal\n2) Hidden")
+        vol_type_dict = {1: "normal",2: "hidden"}
+        vol_type = vol_type_dict[click.prompt("Select", default=1, type=click.IntRange(1, 2), show_choices=False)]
+        click.echo(
+            "Encryption Algorithm:\n"
+            "1) AES\n"
+            "2) Serpent\n"
+            "3) Twofish\n"
+            "4) Camellia\n"
+            "5) Kuznyechik\n"
+            "6) AES(Twofish)\n"
+            "7) AES(Twofish(Serpent))\n"
+            "8) Camellia(Kuznyechik)\n"
+            "9) Camellia(Serpent)\n"
+            "10) Kuznyechik(AES)\n"
+            "11) Kuznyechik(Serpent(Camellia))\n"
+            "12) Kuznyechik(Twofish)\n"
+            "13) Serpent(AES)\n"
+            "14) Serpent(Twofish(AES))\n"
+            "15) Twofish(Serpent)"
+            )
+        encrypt_type_dict = {
+                1: "aes",
+                2: "serpent",
+                3: "twofish",
+                4: "camellia",
+                5: "kuznyechik",
+                6: "aes(twofish)",
+                7: "aes(twofish(serpent))",
+                8: "camellia(kuznyechik)",
+                9: "camellia(serpent)",
+                10: "kuznyechik(aes)",
+                11: "kuznyechik(serpent(camellia))",
+                12: "kuznyechik(twofish)",
+                13: "serpent(aes)",
+                14: "serpent(twofish(aes))",
+                15: "twofish(serpent)"
+                }
+        encrypt_type = encrypt_type_dict[click.prompt("Select", default=1, type=click.IntRange(1, 15), show_choices=False)]
+        click.echo("Hash algorithm:\n1) SHA-512\n2) Whirlpool\n3) BLAKE2s-256\n4) SHA-256\n5) Streebog\n")
+        hash_algo_dict = {1: "sha-512", 2: "whirlpool", 3: "blake2s-256", 4: "sha-256", 5: "streebog"}
+        hash_algo = hash_algo_dict[click.prompt("Select", default=1, type=click.IntRange(1, 5), show_choices=False)]
+        click.echo(
+            "Filesystem:\n"
+            "1) None\n"
+            "2) FAT\n"
+            "3) Linux Ext2\n"
+            "4) Linux Ext3\n"
+            "5) Linux Ext4\n"
+            "6) NTFS"
+            )
+        filesystem_dict = {
+            1: "none",
+            2: "fat",
+            3: "linux ext2",
+            4: "linux ext3",
+            5: "linux ext4",
+            6: "ntfs"
+        }
+        filesystem = filesystem_dict[click.prompt("Select", default=2, type=click.IntRange(1, 6), show_choices=False)]
+        click.echo("\nEnter strong password for encrypting. (You don't have to remember it)")
+        strongp = click.prompt("So set a random password with special characters: ", hide_input=True, confirmation_prompt=True)
```

Comment:
The password set here should be random and strong, and the user doesn't have to remember it. So, set the confirmation_prompt to false here.

Reply:
This is a simple fix, so I will edit it myself.